src/lib/commands/pretty-print-bytes.ts

```diff
@@ -16,7 +16,7 @@ export function prettyPrintBytes({
 		return `${(0).toPrecision(precision)} Byte`;
 	}
 
-	const i = Math.floor(Math.log(bytes) / Math.log(1024));
+	const i = Math.floor(Math.log(bytes) / Math.log(1000));
```

Comment:
wasnt it correct before?

Reply:
Turns out no, the console does / 1000 more often than not, not / 1024 (except i think in some very specific edge cases / pages, I need to re-check their source code some other time)